css-transitions-2/Overview.bs

```diff
@@ -88,7 +88,9 @@ In CSS Transitions Level 2,
 property values are [=transitionable=]
 <em>unless</em> they have an [=animation type=] that is [=not animatable=].
 Values with a [=discrete=] [=animation type=] <em>are</em> [=transitionable=],
-and flip at 50% progress (<var>p</var> = 0.5).
+and flip at 50% progress (<var>p</var> = 0.5) if the 'transition-behavior' is
```

Comment:
To match the terminology for the correct item in the list in css-transforms-1, this should probably refer to the "matching transition behavior".

Reply:
It might be simpler to leave out the "and flip at 50% progress" as this is already defined in web-animations-1. The important part is whether they are transitionable.

If you look at the [definition from css-transitions-1](https://www.w3.org/TR/css-transitions-1/#transitionable):
> When comparing the [before-change style](https://www.w3.org/TR/css-transitions-1/#before-change-style) and [after-change style](https://www.w3.org/TR/css-transitions-1/#after-change-style) for a given property, the property values are transitionable if they have an [animation type](https://drafts.csswg.org/web-animations-1/#animation-type) that is neither [not animatable](https://drafts.csswg.org/web-animations-1/#not-animatable) nor [discrete](https://drafts.csswg.org/web-animations-1/#discrete).

We could update this to something like:

When comparing the [before-change style](https://www.w3.org/TR/css-transitions-1/#before-change-style) and [after-change style](https://www.w3.org/TR/css-transitions-1/#after-change-style) for a given property, the property values are transitionable if:
- They have an [animation type](https://drafts.csswg.org/web-animations-1/#animation-type) that is neither [not animatable](https://drafts.csswg.org/web-animations-1/#not-animatable) nor [discrete](https://drafts.csswg.org/web-animations-1/#discrete), or
- The transition-behavior is allow-discrete and they have an [animation type](https://drafts.csswg.org/web-animations-1/#animation-type) that is [discrete](https://drafts.csswg.org/web-animations-1/#discrete).

Reply:
Thanks rob! I replaced this paragraph with your text